webservices/common/util.py

```diff
@@ -16,3 +16,9 @@ def filter_query(model, query, fields, kwargs):
         column = getattr(model, field)
         query = query.filter(column.in_(value))
     return query
+
+def make_pdf_url(imagenumber):
```

Comment:
+1. cc @LindsayYoung 

Not for this task, but we might want to consolidate the two util files to avoid this sort of mistake in the future.